temperature/crud.py

```diff
@@ -0,0 +1,40 @@
+from fastapi import HTTPException
+
+from sqlalchemy import select, insert
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from temperature import models, schemas
```

Comment:
Ensure that the `models` and `schemas` modules are correctly defined in the `temperature` package. This is crucial for the CRUD operations to function properly.